packages/core/admin/admin/src/components/LeftMenu.tsx

```diff
@@ -135,6 +136,17 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }: LeftMenuProps) =
       <Divider />
 
       <NavSections>
+        {condensed && (
+          <NewNavLink.Link to="/" onClick={() => handleClickOnLink('/')}>
+            <NewNavLink.Tooltip
+              label={formatMessage({ id: 'global.home', defaultMessage: 'Home' })}
+            >
+              <NewNavLink.Icon>
+                <Icon as={House} color="neutral500" />
```

Comment:
House icon looks different than figma, with new icon changes it should be good I guess. also icons size is smaller than expected.

Reply:
The svg is 16x16 like the others svg, I changed the height and width now